### ISSUE TITLE
Index Problem

### ISSUE THREAD

**Issue by xjmps** [xjmps/css-validation-fixes#1]
Morning, 

So while working on my file I had finished everything however it says my website is offline, I'm positive it is because the naming convention on the index is the reason. However, I have fixed the uppercase "I" to a lowercase, but for some reason it will not update on github making it so I can not make it online and when looking on github it says "Index.html", rather than "index.html". Am I missing something or is my file being difficult because I'm lost for how to fix it...

@hellachella

**Comment by hellachella** [comment]
Yes, this is a mac OSX glitch. Simial to HTML validation fixes, you need to rename the index to a different name entirely, commit and push. call it change.html or something

After that, rename it to index.html, commit and push.
